aws-lambda-java-events/src/test/java/com/amazonaws/services/lambda/runtime/events/CloudWatchAlarmEventTest.java

```diff
@@ -0,0 +1,58 @@
+package com.amazonaws.services.lambda.runtime.events;
+
+import com.amazonaws.services.lambda.runtime.serialization.PojoSerializer;
+import com.amazonaws.services.lambda.runtime.serialization.factories.JacksonFactory;
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.skyscreamer.jsonassert.JSONAssert.assertEquals;
+import static org.skyscreamer.jsonassert.JSONCompareMode.STRICT;
+
+public class CloudWatchAlarmEventTest {
```

Comment:
It seems that those tests are not properly run in the CI? See GitHub action output:
```
2024-06-14T21:14:00.1302461Z Running com.amazonaws.services.lambda.runtime.events.CloudWatchAlarmEventTest
2024-06-14T21:14:00.1303887Z Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 sec
```

Reply:
Resolved